src/utils/print.jl

```diff
@@ -131,8 +131,8 @@ end
 
 function Base.show(io::IO, ::MIME"text/plain", ists::Vector{<:InfrastructureSystemsType})
     println(io, summary(ists))
-    for ist in ists
-        println(io, "$(summary(ist))")
+    for i = 1:length(ists)
+        isassigned(ists, i) && println(io, "$(summary(ists[i]))")
```

Comment:
i think we still want to print the undefined values but to print them as `#undef`, i.e. printing a length 3 Vector should show 3 things, it's just that we want to avoid trying to access undefined references. 

e.g.
```julia
     for i = 1:length(ists) 
         if isassigned(ists, i)
             println(io, summary(ists[i]))
         else
            println(io, Base.undef_ref_str)
        end
```

Reply:
Good point, I'll add that. By the way, do you know why format check is failing? I'm not familiar with that.